pages/login.js

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from "react";
+import { useRouter } from "next/router";
+
+const Login = () => {
+  const router = useRouter();
+  const [username, setUsername] = useState("");
+  const [password, setPassword] = useState("");
+
+  const handleSubmit = async (ev) => {
+    ev.preventDefault();
+
+    const data = {
+      username,
+      password,
+    };
+
+    try {
+      const res = await fetch("/api/users", {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify(data),
+      });
+      if (res.ok == true) {
```

Comment:
Nice usage of WebAPI Response.ok checking statuscode ok (200-299)

Reply:
Thanks